app/main.py

```diff
@@ -16,4 +16,13 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+variables = [lucky_number, pi, one_is_a_prime_number, name,
+             my_favourite_films, profile_info, marks, collection_of_coins]
+for data_type in variables:
+    if isinstance(data_type, (int, float, bool, str, tuple)):
+        sorted_variables["immutable"].append(data_type)
+    else:
+        sorted_variables["mutable"].append(data_type)
+
+print(sorted_variables)
```

Comment:
You can explicitly create dictionary with defined keys and values. for loop is unnecessary for this specific task.